src/libraries/Microsoft.PowerFx.Core/Texl/BuiltinFunctionsCore.cs

```diff
@@ -1,219 +1,220 @@
+        public static readonly TexlFunction Coalesce = _library.Append(new CoalesceFunction());
+        public static readonly TexlFunction ColorFade = _library.Append(new ColorFadeFunction());
+        public static readonly TexlFunction ColorFadeT = _library.Append(new ColorFadeTFunction());
+        public static readonly TexlFunction ColorValue = _library.Append(new ColorValueFunction());
+        public static readonly TexlFunction ColorValue_UO = _library.Append(new ColorValueFunction_UO());
+        public static readonly TexlFunction Concat = _library.Append(new ConcatFunction());
+        public static readonly TexlFunction Concatenate = _library.Append(new ConcatenateFunction());
+        public static readonly TexlFunction ConcatenateT = _library.Append(new ConcatenateTableFunction());
+        public static readonly TexlFunction Cos = _library.Append(new CosFunction());
+        public static readonly TexlFunction CosT = _library.Append(new CosTableFunction());
+        public static readonly TexlFunction Cot = _library.Append(new CotFunction());
+        public static readonly TexlFunction CotT = _library.Append(new CotTableFunction());
+        public static readonly TexlFunction Count = _library.Append(new CountFunction());
+        public static readonly TexlFunction CountA = _library.Append(new CountAFunction());
+        public static readonly TexlFunction CountIf = _library.Append(new CountIfFunction());
+        public static readonly TexlFunction CountRows = _library.Append(new CountRowsFunction());
+        public static readonly TexlFunction CountRows_UO = _library.Append(new CountRowsFunction_UO());
+        public static readonly TexlFunction Date = _library.Append(new DateFunction());
+        public static readonly TexlFunction DateAdd = _library.Append(new DateAddFunction());
+        public static readonly TexlFunction DateAddT = _library.Append(new DateAddTFunction());
+        public static readonly TexlFunction DateDiff = _library.Append(new DateDiffFunction());
+        public static readonly TexlFunction DateDiffT = _library.Append(new DateDiffTFunction());
+        public static readonly TexlFunction DateTimeValue = _library.Append(new DateTimeValueFunction());
+        public static readonly TexlFunction DateTimeValue_UO = _library.Append(new DateTimeValueFunction_UO());
+        public static readonly TexlFunction DateValue = _library.Append(new DateValueFunction());
+        public static readonly TexlFunction DateValue_UO = _library.Append(new DateValueFunction_UO());
+        public static readonly TexlFunction Day = _library.Append(new DayFunction());
+        public static readonly TexlFunction Degrees = _library.Append(new DegreesFunction());
+        public static readonly TexlFunction DegreesT = _library.Append(new DegreesTableFunction());
+        public static readonly TexlFunction Distinct = _library.Append(new DistinctFunction());

```

Comment:
>_library [](http://example.com/codeflow?start=55&length=8)

We may need to register this differently to ensure that it doesn't conflict with PowerApps' definition.

Reply:
Will wait for your advice in the new PR.